### PR TITLE
Add pspell extension

### PR DESCRIPTION
**Pull request by alexmace**

### CODE REVIEW

File: spec/fixtures/Puppetfile [hunk 1]
```diff
@@ -12,3 +12,4 @@ mod "mysql",       "1.1.3", :github_tarball => "boxen/puppet-mysql"
 mod "postgresql",  "2.0.1", :github_tarball => "boxen/puppet-postgresql"
 mod "nginx",       "1.4.2", :github_tarball => "boxen/puppet-nginx"
 mod "module_data", "0.0.3", :github_tarball => "ripienaar/puppet-module-data"
+mod "aspell",	   "0.0.1", :github_tarball => "namesco/puppet-aspell"
```
Comment (mattheath) [inline spec/fixtures/Puppetfile]: super minor, can the version string be aligned please

**Comment by alexmace** [comment]
Version string is now aligned as requested.

**Comment by mattheath** [comment]
I don't know much (actually, anything) about aspell/pspell, but the module looks good. Thanks! :smile:

**Comment by jacobbednarz** [comment]
@alexmace it looks like this has caused some issues with the master build - https://travis-ci.org/boxen/puppet-php/jobs/162044458. do you mind having a look?

**Pull request merged**